pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -1,18 +1,22 @@
 { stdenv, fetchgit, zlib, libpng, qt4, pkgconfig
-, withGamepads ? true, SDL # SDL is used for gamepad functionality
+, withGamepads ? true, SDL ? null # SDL is used for gamepad functionality
```

Comment:
Well, just cosmetic things, if you prefer to think that way :). 
By the way, I tested it on SDL2, too. I will change the main all-packages script to reflect this.